src/mlagility/common/filesystem.py

```diff
@@ -23,10 +24,7 @@
 BUILD_MARKER = ".mlabuild"
 
 # Locate the models directory
-# This file is located in MLAGILITY_INSTALL/src/mlagility/common/
-# The models directory is located at MLAGILITY_INSTALL/models
-# This relative positioning is protected by the "test_000_models_dir" in test/unit.py
-MODELS_DIR = pathlib.Path(__file__).parents[3] / "models"
+MODELS_DIR = importlib.util.find_spec("mlagility_models").submodule_search_locations[0]
```

Comment:
Got the following error when trying this by cloning and pip installing:

```
Traceback (most recent call last):
  File "/net/home/dhnoronha/miniconda3/envs/atlas/bin/benchit", line 33, in <module>
    sys.exit(load_entry_point('mlagility', 'console_scripts', 'benchit')())
  File "/net/home/dhnoronha/miniconda3/envs/atlas/bin/benchit", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/net/home/dhnoronha/miniconda3/envs/atlas/lib/python3.8/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/net/home/dhnoronha/miniconda3/envs/atlas/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/net/home/dhnoronha/mlagility/src/mlagility/__init__.py", line 3, in <module>
    from .api.script_api import benchmark_script
  File "/net/home/dhnoronha/mlagility/src/mlagility/api/script_api.py", line 10, in <module>
    import mlagility.cli.slurm as slurm
  File "/net/home/dhnoronha/mlagility/src/mlagility/cli/slurm.py", line 7, in <module>
    import mlagility.common.filesystem as filesystem
  File "/net/home/dhnoronha/mlagility/src/mlagility/common/filesystem.py", line 27, in <module>
    MODELS_DIR = importlib.util.find_spec("mlagility_models").submodule_search_locations[0]
AttributeError: 'NoneType' object has no attribute 'submodule_search_locations'
```
When changing it to `MODELS_DIR = pathlib.Path(__file__).parents[3] / "models"` things work as expected.

Reply:
pip installing again solves it!